Sources/tasks/Likes-Notify.php

```diff
@@ -74,6 +76,11 @@ public function execute()
 		if ($author == $this->_details['sender_id'])
 			return true;
 
+		// If the person who sent the notification is on this person's ignore list, do nothing.
+        if (!empty($ignored_members) && in_array($this->_details['sender_id'], $ignored_members)) {
+            return true;
+        }
+
```

Comment:
Thank you, been having a hell of a time with my IDE on this. I think it is the VIM plugin for it.